examples/CPMG.rst

```diff
@@ -0,0 +1 @@
+.. image:: CPMG_ppg.jpg
```

Comment:
don't make a separate file -- put this right in the docstring.

Reply:
done